src/status_im/hardwallet/fx.cljs

```diff
@@ -46,3 +43,55 @@
 (re-frame/reg-fx
  :hardwallet/generate-and-load-key
  card/generate-and-load-key)
+
+(re-frame/reg-fx
+ :hardwallet/unblock-pin
+ card/unblock-pin)
+
+(re-frame/reg-fx
+ :hardwallet/verify-pin
+ card/verify-pin)
+
+(re-frame/reg-fx
+ :hardwallet/change-pin
+ card/change-pin)
+
+(re-frame/reg-fx
+ :hardwallet/unpair
+ card/unpair)
+
+(re-frame/reg-fx
+ :hardwallet/delete
+ card/delete)
+
+(re-frame/reg-fx
+ :hardwallet/unpair-and-delete
+ card/unpair-and-delete)
+
+(re-frame/reg-fx
+ :hardwallet/persist-pairing
+ (fn [pairing]
+   (.. js-dependencies/react-native
+       -AsyncStorage
```

Comment:
Why AsyncStorage and not realm?

Reply:
@jeluard It was easier to start with while i was experimenting with this. Also, this data is stored on application level and not tied to particular account (it couldn't be stored in  "account" realm). But i think i could use "core" realm for storing this as well.

Reply:
Sounds like it would make sense to align with realm? Unless there is a specific reason not to use it.

Reply:
@jeluard yes, i will switch to using realm

Reply:
@jeluard i switched to using account realm to store pairing data, but also temporary left async storage as we need that for testing purposes. We don't have way to login with keycard account for now, so for testing we use regular account to access Keycard Settings screen and we need to access pairing data (which is accessible from async storage).

I will remove using async storage after we implement login with keycard.